docs/character_encoding_override.rst

```diff
@@ -0,0 +1,32 @@
+Character Encoding Override
+===========================
+
+When yamllint reads a file, it will try to automatically detect that file’s
```

Comment:
Idea:
```suggestion
When yamllint reads a file (either configuration or a file to lint), it will try
to automatically detect that file’s …
```
(this goes along with my next comment ↓)

Reply:
I like that suggestion. I incorporated your suggestion into the version of the pull request that I just pushed, but I used slightly different wording (I wanted to make the sentence slightly easier to understand and sound more natural). What do you think about the wording that I used?

Reply:
It's perfect, thanks! (I'm not a native English speaker so such enhancements are always welcome :slightly_smiling_face:)